modules/apps/password-policies-admin/password-policies-admin-web/src/main/java/com/liferay/password/policies/admin/web/internal/display/context/PasswordPolicyDisplayContext.java

```diff
@@ -144,12 +142,29 @@ public PasswordPolicyDisplayContext(
 		return navigationItems;
 	}
 
-	public boolean hasAssignMembersPermission() {
-		long passwordPolicyId = ParamUtil.getLong(_request, "passwordPolicyId");
+	private PasswordPolicy _getPasswordPolicy() {
+		if (_passwordPolicy != null) {
+			return _passwordPolicy;
+		}
+
+		_passwordPolicy = PasswordPolicyLocalServiceUtil.fetchPasswordPolicy(
+			_getPasswordPolicyId());
+
+		return _passwordPolicy;
+	}
+
+	private long _getPasswordPolicyId() {
+		if (_passwordPolicyId != 0L) {
```

Comment:
In a case like this, we should use a [boxed type](https://stackoverflow.com/questions/1418296/what-does-it-mean-to-say-a-type-is-boxed), meaning that `_passwordPolicyId` will be of type `Long` rather than `long`.  That way we can do a null check just like we do for `_getPasswordPolicy`.  This can be especially important when `0` is a valid value (which is the case when we're adding a password policy).